Form-Controls/index.html

```diff
@@ -6,23 +6,74 @@
   <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@600&family=Quicksand:wght@300&display=swap" rel="stylesheet">
   <link rel="stylesheet" href="styles.css">
 </head>
 <body>
   <header>
-    <h1>Product Pick</h1>
+    <div>
+      <h1>Product Pick</h1>
+    </div>
   </header>
-  <main>
-  <form>
-    <!-- write your html here-->
-    <!-- try writing out the requirements first-->
-  </form>
-
-  </main>
+  <section>
+    <div class="section__container">
+      <form action="">
+        <fieldset>
+          <div class="name__wrapper">
+            <div>
+                <label for="firstName">First Name *</label>
+                <input type="text" name="" id="firstName" required pattern="[A-Za-z]{3}" title="Must contain at least 3 letters">
+            </div>
+            <div>
+                <label for="lasttName">Last Name *</label>
+                <input type="text" name="" id="lasttName" required pattern="[A-Za-z]{3}" title="Must contain at least 3 letters">
+            </div>
+        </div>
+        <div>
+            <div class="form__wrapper">
+                <label for="email">Email *</label>
+                <input type="email" id="email" required title="Email must be valid example@mail.com">
+            </div>
+        </div>
+        <div class="form__wrapper">
+            <label for="color">Choose colour</label>
+            <select name="color" id="color">
+              <option value="" disabled selected>Choose colour</option>
+              <option value="black">Black</option>
+              <option value="white">White</option>
+              <option value="violet">Violet</option>
+            </select>
+        </div>
+        <div class="form__wrapper">
+            <label for="size">Choose size</label>
+            <select name="size" id="size">
+              <option value="" disabled selected>Choose size</option>
+              <option value="XS">XS</option>
+              <option value="S">S</option>
+              <option value="M">M</option>
+              <option value="L">L</option>
+              <option value="XL">XL</option>
+              <option value="XXL">XXL</option>
+            </select>
+        </div>
+        <div class="form__wrapper">
+          <label for="delivery">Choose delivery date</label>
+          <input type="date" id="delivery" min="2023-02-14" max="2023-03-14" required>
+        </div>
+        </fieldset>
+        <div class="form__btn">
+          <button class="contact__btn" type="submit">SEND</button>
+      </div>
+      </form>
+    </div>
+  </section>
   <footer>
-    <!-- change to your name-->
-    <h2>By HOMEWORK SOLUTION</h2>
-  </footer>
+    <div class="footer" id="contact">
+        <p>Join us now</p>
+      <span>© by Maksim Lukianenko</span>
+      <a href="https://codeyourfuture.io/"><img class="footer__img" src="/Form-Controls/files/cyf_brand.png" alt="CYF"></a>
```

Comment:
I love that you added this! 😄 

I have to break it to you that you are distorting the aspect ratio. 🟥  Take a look in Lighthouse to see how to fix things!

Reply:
I corrected it using object-fit prop.